modular/signer/signer_client.go

```diff
@@ -442,3 +443,66 @@ func (client *GreenfieldChainSignClient) CreateGlobalVirtualGroup(ctx context.Co
 	client.operatorAccNonce = nonce + 1
 	return txHash, nil
 }
+
+func (client *GreenfieldChainSignClient) CompleteMigrateBucket(ctx context.Context, scope SignType, completeMigration *storagetypes.MsgCompleteMigrateBucket) ([]byte, error) {
```

Comment:
refer to: https://github.com/bnb-chain/greenfield/pull/284/files#diff-cc3cc646a27b4bedb38d230d96b74c3dd122f357b7f82e6ed80392a5f3c5763cR1543, Do we also need an additional signature interface.

Reply:
`migrationBucketSignHash := storagetypes.NewSecondarySpMigrationBucketSignDoc(bucketInfo.Id, dstPrimarySP.Info.Id, gvgMapping.SrcGlobalVirtualGroupId, gvgMapping.DstGlobalVirtualGroupId).GetBlsSignHash()`